src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/slicing/identifier/ReverseIdentifierFactory.java

```diff
@@ -17,6 +17,13 @@ public long getCountryOsmIdentifier(final long countryCodeAndWaySectionedIdentif
                 countryCodeAndWaySectionedIdentifier / AbstractIdentifierFactory.IDENTIFIER_SCALE);
     }
 
+    public long getFirstAtlasIdentifier(final long countryCodeAndWaySectionedIdentifier)
```

Comment:
Method comments would be useful to include here.

Reply:
Good idea, let me add them for the entire class.